main.cpp

```diff
@@ -54,48 +54,76 @@ Purpose:  This project will show you the difference between member functions and
 
 #include <iostream>
 #include <string>
-struct T
+struct SoccerBall
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
-    //2
+    SoccerBall(float v, const char* brand): value(0), name("")
+    {
+        value = v;
+        name = brand;
+    }   //1
```

Comment:
Use the member initializer list correctly.
pass the constructor arguments to the member variables
- [ ] revise

Reply:
Is this odd because I am using the constructor initializer list to values other than the parameters passed into the constructor?